docs.md

```diff
@@ -1,23 +1,396 @@
+         }
+         },
+         {
+         "command": "input_prop_in",
+         "properties": {
+            "input_property": "company",
+            "datasource_name": "items",
+            "datasource_loop_variable": "name"
+         }
+         },
+         {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "request_method",
+            "value": "GET"
+         }
+         }
+      ],
+      [
+         {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "request_path",
+            "value": [
+               "v1",
+               "collections",
+               "obs",
+               "*"
+            ]
+         }
+         },
+         {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "company",
+            "value": "geobeyond"
+         }
+         },
+         {
+         "command": "input_prop_in_as",
+         "properties": {
+            "datasource_name": "items",
+            "datasource_loop_variables": [
+               "name",
+               "groupname"
+            ],
+            "data_input_properties": [
+               "preferred_username",
+               "groupname"
+            ]
+         }
+         }
+      ],
+      [
+         {
+         "command": "allow_full_access",
+         "properties": {
+            "input_property": "groupname",
+            "value": "EDITOR_ATAC"
+         }
+         }
+      ],
+      [
+         {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "groupname",
+            "value": [
+               "v1",
+               "collections",
+               "test-data"
+            ]
+         }
+         },
+         {
+         "command": "allow_full_access",
+         "properties": {
+            "input_property": "name",
+            "value": "admin"
+         }
+         }
+      ],
+      [
+         {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "request_path",
+            "value": [
+               "v1",
+               "collections",
+               "lakes" ]
+            }
+         }
+      ]
+   ]}
+
+   ```
+ ###  - GET `/policies/` 🗝
+This route is used to get all policies that have been created. The response will be a list of all policies that have been created by a certain user, and contains all the associating rules with the policy <br />
+
+```py3
+List[Policy]   
+   ```
+
+   ###  - GET PUT DELETE `/policies/{policy_name}` 🗝
+   GET - This request method is used to get a specific policy by name. The response will be a policy object conforming to the pydantic model `Policy`. <br />
+   PUT - This request method is used to update a specific policy by name. The response will be a policy object conforming to the pydantic model `Policy`. <br />
+   DELETE - This request method is used to delete a specific policy by name
```

Comment:
They all need to show:

- Sample requests and responses
- What happens when a policy isn't found?